.sfdevrc.json

```diff
@@ -2,7 +2,6 @@
   "script": {
     "prepare": "sf-install && yarn compile && yarn lint",
     "pretest": "echo disable # sf-compile-test",
-    "test": "echo disable # sf-test",
```

Comment:
Was this on purpose?

Reply:
@RodEsp Yes, cli now has unit tests to run.

Reply:
But this completely removes the test script.

Did you mean to leave this?
```
"test": "sf-test"
```

Reply:
@RodEsp removing this means the change won't be made.

See https://github.com/salesforcecli/cli/blob/phale/W-10536395/package.json#L141